topics/cluster-tutorial.md

```diff
@@ -363,110 +374,128 @@ world conditions. It is not very helpful to see what happens while nobody
 is writing to the cluster.
 
 This section explains some basic usage of
-[redis-rb-cluster](https://github.com/antirez/redis-rb-cluster) showing two
-examples. 
-The first is the following, and is the
-[`example.rb`](https://github.com/antirez/redis-rb-cluster/blob/master/example.rb)
-file inside the redis-rb-cluster distribution:
-
-```ruby
-require './cluster'
-
-if ARGV.length != 2
-    startup_nodes = [
-        {:host => "127.0.0.1", :port => 7000},
-        {:host => "127.0.0.1", :port => 7001}
-    ]
-else
-    startup_nodes = [
-        {:host => ARGV[0], :port => ARGV[1].to_i}
-    ]
-end
-
-rc = RedisCluster.new(startup_nodes,32,:timeout => 0.1)
-
-last = false
-
-while not last
-    begin
-        last = rc.get("__last__")
-        last = 0 if !last
-    rescue => e
-        puts "error #{e.to_s}"
-        sleep 1
-    end
-end
-
-((last.to_i+1)..1000000000).each{|x|
-    begin
-        rc.set("foo#{x}",x)
-        puts rc.get("foo#{x}")
-        rc.set("__last__",x)
-    rescue => e
-        puts "error #{e.to_s}"
-    end
-    sleep 0.1
-}
+[Valkey GLIDE for Node.js](https://github.com/valkey-io/valkey-glide/tree/main/node), an official
+Valkey client library, available in numerous languages, showing a practical example application in Node.js.
```

Comment:
This is slightly confusing. GLIDE is 'available in numerous languages' not Valkey GLIDE for Node.js.

I might rephrase this entire passage.